src/Compilers/CSharp/Portable/Binder/Binder_Lookup.cs

```diff
@@ -1465,7 +1465,7 @@ private static bool IsInstance(Symbol symbol)
                 case SymbolKind.Property:
                 case SymbolKind.Method:
                 case SymbolKind.Event:
-                    return !symbol.IsStatic;
+                    return symbol.RequiresInstanceReceiver();
```

Comment:
>RequiresInstanceReceiver [](start = 34, length = 24)

compiler layer changes should come with compiler layer validation as well. Thanks

Reply:
Done.

Though the new test does highlight that the naming of `LookupStaticMembers` is misleading: even before this change, it returned local variables too. Maybe at least its documentation should be updated?

Reply:
Done.

Though the new test does highlight that the naming of `LookupStaticMembers` is misleading: even before this change, it returned local variables too. Maybe at least its documentation should be updated?